### ISSUE TITLE
i18n routes are suffixed

### ISSUE THREAD

**Issue by cannap** [victorgarciaesgi/nuxt-typed-router#88]
Hi,
i alway get this output  with defaultLanguage: 'en'

```
export type RoutesNamesList =
  | 'dashboard___en'
  | 'index___en'
  | 'me___en'
  | 'me-settings___en'
  | 'signin___en'
export type WithoutBracket<T extends string> = T extends `:${string}`
  ? never
  : T
```
![image](https://user-images.githubusercontent.com/1394151/233790626-7d6f6ca4-4dda-446d-a1e4-1f78f5056530.png)

```js
 i18n: {
    // strategy: 'prefix_except_default',
    lazy: true,
    defaultLocale: 'en',
    langDir: 'locales',
    locales: [
      { code: 'en', file: 'en.json', iso: 'en-US', name: 'English' },
      { code: 'de', file: 'de.json', iso: 'de-DE', name: 'German' }
    ]
  
  },
  ```
  

when i remove defaultLanguage i get 
```export type RoutesNamesList =
  | 'dashboard___en'
  | 'dashboard___de'
  | 'index___en'
  | 'index___de'
  | 'me___en'
  | 'me-settings___en'
  | 'me___de'
  | 'me-settings___de'
  | 'signin___en'
  | 'signin___de'
export type WithoutBracket<T extends string> = T extends `:${string}`
  ? never
  : T
  ```

"nuxt-typed-router": "3.1.4",
"@nuxtjs/i18n": "8.0.0-beta.11",
edit:

i updated to 3.1.5 same 
this removes the ts error but i its wrong 
`  <NuxtLink :to="localePath({ name: 'me-settings___en' })">Settings</NuxtLink>`

thank you

**Comment by victorgarciaesgi** [comment]
@cannap is it working with the beta version?

**Comment by riddla** [comment]
Fixed it for me with the `3.2.0` release 👍
